src/components/dev-hub/article-share-footer.js

```diff
@@ -12,7 +12,7 @@ import FacebookIcon from './icons/facebook-icon';
 import TwitterIcon from './icons/twitter-icon';
 
 const ArticleShareArea = styled('div')`
-    border-top: 1px solid ${colorMap.greyDarkTwo};
+    border-top: 1px solid ${darkTheme.colorMap.greyDarkTwo};
```

Comment:
Ah my fault, I thought the PR I linked had one or two components using the theme but alas they did not. Fun fact: we don't need to actually import and use `darkTheme` for any of these styled components.

What happens is in `layout.js` we set the theme in a component called `ThemeProvider`. This allows us to access the active theme (which may change from `darkTheme`) in our styled component declaration as such:

`border-top: 1px solid ${({theme}) => theme.colorMap.greyDarkTwo}`

Seems like magic, right? That is because when we declare the theme in `layout.js`, it gets passed to all child styled components so they can use just as any other prop.

Again sorry I thought that PR did one as such, but let's follow this pattern so the proper theme is being used, thanks!

Reply:
You can see this in action [in the docs](https://emotion.sh/docs/theming)

Reply:
Thanks and no worries at all. I realized I didn't do it the right way after creating the PR (which is why I converted it into a draft). I'm on it!